internal/distro/rhel90/distro.go

```diff
@@ -460,10 +459,6 @@ func New() distro.Distro {
 	return newDistro(defaultName, modulePlatformID, ostreeRef)
 }
 
-func NewRHEL86() distro.Distro {
-	return newDistro(rhel86Name, modulePlatformID, ostreeRef)
```

Comment:
Couldn't this be 9.1 alias instead?

Reply:
Ah, right, a good thing for a follow-up.